app/controllers/insights_cloud/ui_requests_controller.rb

```diff
@@ -9,7 +9,7 @@ def forward_request
       begin
         @cloud_response = ::ForemanRhCloud::InsightsApiForwarder.new.forward_request(
           request,
-          params.require(:path),
+          request.original_fullpath.match(%r{^/insights_cloud/(?<path>[^?]*)})[:path],
```

Comment:
Rails "helpfully" unescapes `params.require(:path)`, but we need to keep it escaped here so it doesn't get forwarded with the bad pipe character.

Reply:
How would that resolve it?